tests/codeception/acceptance/OpportunityContentCest.php

```diff
@@ -37,6 +37,17 @@ public function testOpportunityContentCreation(\AcceptanceTester $I) {
     $I->canSee('Lorem Ipsum');
   }
 
+  /**
+   * The importer should bring in some content.
+   */
+  public function testImporter(\AcceptanceTester $I) {
+    $I->logInWithRole('administrator');
+    $I->amOnPage('/admin/structure/migrate/manage/opportunities/migrations');
+    $I->canSee('solo_opportunities');
+    $total_items = $I->grabTextFrom('table tbody td:nth-child(4)');
+    $I->assertGreaterOrEquals(1, (int) $total_items);
```

Comment:
Shouldn't this test run the import first? Or am I missing where it does that?

Reply:
I tried that but the migration takes too long and it runs out of execution time. we'll probably need to work on that later

Reply:
Hm. Is CircleCI running this test via CodeCeption? 

I see the value of having the test written and included, even if it's not running.

Reply:
i added commented out steps for the drush mim execution.